src/lib/events/__tests__/click.test.js

```diff
@@ -70,6 +70,7 @@ describe('click event delegate', function () {
     var defaultPrevented;
     var link;
     var spanWithinLink;
+    var triggerSpy;
```

Comment:
small cleanup that makes a spy for every test rather than using a new plain function as before

Reply:
That was intentional. At some point, Aaron and I agreed that we will use spies only when they are used in a test. Not sure what his stance is right now on this issue.

Reply:
Whatever you two want to do works for me.